### DISCUSSION TITLE
problem during installation

### DISCUSSION THREAD

**Discussion by kuwaity33** [inspircd/inspircd#2127] [Support] [answered]
Hello, there is a problem during installation. If you activate during installation, modules this error appears.
make: *** [GNUmakefile:212: target] Error 2
[cool@kw inspircd-4.3.0]$ cd
Error: unable to find the libargon2 compiler flags for argon2!

**Comment by sadiepowell** [comment, marked as answer]
You have enabled the argon2 module without having the dependencies installed.

You should either install the dependencies or run

    rm src/modules/m_argon2.cpp

**Comment by kuwaity33** [comment]
Heloo.  Sadie
I could not connect the services. I do not know why? 
anope-2.1.3 + inspircd v4.3.0
Activated spanningtree 
I modified the files and these errors appear?
./anope --nofork

 Anope 2.1.3, build #1, compiled 10:16:00 Nov  4 2024
 Using configuration file conf/anope.conf
 Inheriting commands and privs from Helper to Services Operator
Inheriting commands and privs from Services Operator to Services Administrator
Configuration option options:seed should be set. It's for YOUR safety! Remember that!
SERVER: services.cool.net (Anope IRC Services) has connected to the network (uplinked to no uplink)
 Anope 2.1.3 starting up
 Loading modules...
 Using IRCd protocol inspircd
 Loading databases...
 Databases loaded
 Attempting to connect to uplink #1 88.78.142.958 (88.78.142.958/7000) with protocol InspIRCd 3+
 Unable to connect to uplink #1 (88.78.142.958:7000): Connection refused
 Unable to connect to any uplink

What is the solution? Thank you.

**Reply by sadiepowell** [reply]
You probably haven't added a listener for that port and/or haven't loaded the spanningtree module.
